internal/nodes/avi_model_nodes.go

```diff
@@ -1017,9 +1017,14 @@ func (v *AviHttpPolicySetNode) CalculateCheckSum() {
 	var checksum uint32
 	for _, hpp := range v.HppMap {
 		sort.Strings(hpp.Path)
-		sort.Strings(hpp.Host)
-		checksum = checksum + utils.Hash(utils.Stringify(hpp))
-
+		checksum += utils.Hash(utils.Stringify(hpp.Name)) +
```

Comment:
you can copy hpp into a new variable and remove Host from that.

Reply:
I have made changes as per the comment.